### PR TITLE
Bug 1181415 - [Contacts] Convert switches to use gaia-switch component

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1181415

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8a1e2bc5143f763567bcc3253833b582c5592b2b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d899bfbb3512281bd4821f7247a6ebb16a98c40d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f9f1c261a5d6785ce9b797676d01c7803a820ccb

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=171e04ee030728bcc9588dce6cd51e62fcd171c5

File: apps/communications/contacts/index.html [hunk 1]
```diff
@@ -77,6 +77,7 @@
     <script defer src="/shared/elements/config.js"></script>
     <script defer src="/shared/js/component_utils.js"></script>
     <script defer src="/shared/elements/gaia_subheader/script.js"></script>
+    <script defer src="/shared/elements/gaia_switch/script.js"></script>
```
Comment (arcturus) [inline apps/communications/contacts/index.html]: Can we remove this script from the critical path and add it to the dependencies when we load the settings panel?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cf92b4058897fc3b81aa2cacc64667f7f4a8cc61

**Pull request merged**